requirements.txt

```diff
@@ -1,60 +1,24 @@
+# TODO: switch from -d to -c and unpin jinja2, bokeh once katsdpdockerbase
+# updated.
+-d https://raw.githubusercontent.com/ska-sa/katsdpdockerbase/master/docker-base-build/base-requirements.txt
+
 ansicolors
-appdirs                   # via katsdpsigproc
-argparse                  # via python-casacore
 astropy
 bokeh==2.0.2              # katsdpdockerbase currently provides only 1.x
 botocore                  # for S3 authentication in katdal
-certifi                   # via requests
 cffi==1.14.1
-chardet                   # via requests
-cityhash                  # via katdal
 contextvars==2.4
-cycler                    # via matplotlib
-dask                      # via katdal
-decorator                 # via pycuda, katsdpsigproc
-docutils                  # via botocore
-ephem                     # via pyephem
-future                    # via katdal, katpoint
-h5py                      # via katdal, katsdpmodels
```

Comment:
`preprocess` and `primary_beam` uses `h5py` directly.

Reply:
Thanks, fixed!